catalyst/metrics/functional/_cmc_score.py

```diff
@@ -50,4 +50,45 @@ def cmc_score(
     return cmc_score_count(distances, conformity_matrix, topk)
 
 
-__all__ = ["cmc_score_count", "cmc_score"]
+def masked_cmc_score(
```

Comment:
could you please also add it to the `catalyst/metrics/functional/__init__.py` and the [docs](https://github.com/catalyst-team/catalyst/blob/master/docs/api/metrics.rst)?